modules/visual_script/visual_script.cpp

```diff
@@ -63,6 +63,13 @@ void VisualScriptNode::ports_changed_notify() {
 	default_input_values.resize(MAX(default_input_values.size(), get_input_value_port_count())); //let it grow as big as possible, we don't want to lose values on resize
 
 	emit_signal("ports_changed");
+
+	int dvc = get_input_value_port_count();
```

Comment:
What does `dvc` stand for? I'm reading this and would have no idea what `dvc` stands for, even with the hint that it's some form of count.

Reply:
In fact, the dvc variable is only used in the loop count. Could put  get_input_value_port_count() there.